app/components/LinksGroup/LinksGroup.tsx

```diff
@@ -97,18 +97,29 @@ const LinksGroup = ({
   size,
   slug,
 }: LinksGroupProps) => {
-  const [opened, setOpened] = useState(initiallyOpened || false)
-
   const location = useLocation()
-  const isActive = location.pathname.includes(slug)
+  const [opened, setOpened] = useState(() => {
+    if (initiallyOpened) {
+      return true
+    }
+
+    if (location.pathname.includes(slug)) {
+      return true
+    }
+
+    return false
+  })
 
   const theme = useMantineTheme()
 
+  const splittedPathname = location.pathname.split("/")
+  const isActive = splittedPathname[splittedPathname.length - 1].includes(slug)
```

Comment:
I think there's a typo here? 
It should be `splittPathname`

Reply:
if it was a function it would be `splitPathname`, since it's a variable that has been declared (in the past) it means it would be `splittedPathname` because it's past tense, but I just realized it's an archaic and nonstandard word :/